### ISSUE TITLE
The demo `lv_demo_music `shows an exception when the color is 24 bits or 16 bits

### ISSUE THREAD

**Issue by IAMMX** [kisvegabor/lvgl#5173]
### LVGL version

v9.0.0

### What happened?

The demo `lv_demo_music `shows an exception when the color is 24 bits or 16 bits, and is displayed normally when it is 32 bits. May I ask if anyone can repair it?
### #define LV_COLOR_DEPTH 32
![32](https://github.com/lvgl/lvgl/assets/150422110/110c82fe-6836-4eb4-98e7-72aa5749add6)
### #define LV_COLOR_DEPTH 24
![24](https://github.com/lvgl/lvgl/assets/150422110/743e088e-f177-457c-a74f-9fd6b2164bca)
### #define LV_COLOR_DEPTH 16
![16](https://github.com/lvgl/lvgl/assets/150422110/1168f697-dbf2-4ee3-ae80-dc6efeb2e099)

**Comment by IAMMX** [comment]
@kisvegabor May I ask if this issue can be addressed?

**Comment by IAMMX** [comment]
@kisvegabor Can you look at this issue?

**Comment by kisvegabor** [comment]
Hi,

Probably I will remove the music player demo from v9 because:
1. It's a shame but the original assets were lost and I can covert new C array from them for v9
2. Due to some changes in v9, it would be quite complicated to implement the jumping strides.

**Comment by XuNeo** [comment]
> It's a shame but the original assets were lost and I can covert new C array from them for v9

That's too unfortunate.
The LVGLImage.py can convert LVGL bin image back to PNG if you want.

```python

def test():
    f = "pngs/cogwheel.RGB565A8.png"
    img = LVGLImage().from_bin("output/cogwheel.RGB888.bin")
    img.to_png("output/cogwheel.RGB888.png.png")  # convert back to png
```

**Comment by kisvegabor** [comment]
We have only the C arrays. :slightly_frowning_face: 

I could update the C assets by hand too, mainly just refactoring, but RGB565A8 is not supported in SW render anymore, so ARGB8888 needed to be used instead.

The difficulty in re-implementing the jumping strides is bothering me more. The correct solution would be to use vector graphics for those trapezoids but I'm afraid it would be too slow.

**Comment by IAMMX** [comment]
It's a pity if the demo of the music player is gone, what a good demo!😭Could you please find some other pictures from the Internet to replace the missing pictures.

**Comment by kisvegabor** [comment]
Seeing that it's important for you both, I'll give it an other try. Probably it can be updated only after the release of v9 though. :slightly_frowning_face:

**Comment by lhdjply** [comment]
fix it in #5291, let's continue it
